quickstarts/biztalk360/dashboards/biztalk360.json

```diff
@@ -0,0 +1,465 @@
+{
+    "name": "BizTalk360",
+    "description": null,
+    "permissions": "PUBLIC_READ_WRITE",
```

Comment:
```suggestion
```

Reply:
This field is generated when exporting a dashboard's json from New Relic, but you don't need it in a quickstart.